baseline/OSPS-VM.yaml

```diff
@@ -94,4 +94,48 @@ criteria:
       status check passes before changes can be
       merged.
     control_mappings: # TODO
+
+  - id: OSPS-VM-05
+    maturity_level: 1
+    criterion: |
+      The project will publish contacts and process
+      for reporting vulnerabilities discovered within
+      the software code or dependencies.
+    rationale: |
+       # TODO 
+    details: |
+       Create a security.md file that contacts security 
+       contacts for the project and provide project's
+       vulnerability handling process.
```

Comment:
```suggestion
       Create a security.md (or similarly-named) file that contains security 
       contacts for the project and provide project's
      process for handling vulnerabilities in the project or dependencies.
```

Reply:
To avoid parentheticals and give a second example... I'd suggest...

```suggestion
       Publish information in a well-known location, such as
       security.md or security-insights.yml, to list security 
       contacts for the project or repo and outline the
       process for handling vulnerabilities in the project
       or its dependencies.
```

Reply:
I like Eddie's suggestion better than mine.